src/pages/registration/DescriptionPanel.tsx

```diff
@@ -60,10 +60,11 @@ export const DescriptionPanel = () => {
             />
           )}
         </Field>
-        <Field name={`${DescriptionFieldNames.AlternativeTitles}.und`}>
+        <Field name={DescriptionFieldNames.AlternativeTitles}>
           {({ field }: FieldProps<string>) => (
             <>
               <Button
+                sx={{ height: 'fit-content', mt: '0.5rem' }}
```

Comment:
er `mt` lagt til for å sentrere knappen? i så fall kan vi kanskje prøve `alignItems: center` på grid eller noe?
Hvis det er for å sentrere vil det være mer tilfeldig nå. Det kommer til å bli litt irriterende feil når feilmelding popper opp og ut, men det får så være tenker jeg

Reply:
funka med en `alignSelf: 'center` på knappen👍